contracts/HybridCustody.cdc

```diff
@@ -1025,36 +1070,25 @@ pub contract HybridCustody {
         /// assumes ownership of an account to guarantee that the previous owner doesn't maintain admin access to the
         /// account via other AuthAccount Capabilities.
         ///
-        pub fun rotateAuthAccount() {
+        access(Owner) fun rotateAuthAccount() {
```

Comment:
The reason this is being done is so that no one can:

1. Make a new Account link
2. "give" ownership to someone (maybe in exchange for something)
3. once new owner thinks they have ownership, use new account link from step 1 and revoke their ownership again

So long as we permit using the interface types over concrete types, nothing would prevent someone from defining their own version of the appropriate resources to ensure things work the way they need